clang/lib/Driver/OffloadBundler.cpp

```diff
@@ -1365,6 +1367,20 @@ Error OffloadBundler::BundleFiles() {
   return Error::success();
 }
 
+bool CheckTripleIsExcluded(
```

Comment:
Can we make this a member function, so we do not need to pass ExcludedTargetNames along? Nit

Reply:
Yeah.